vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,228 @@
+---
+title: vcpkg_make_configure
+description: Use vcpkg_make_configure to configure a Make-based project.
+ms.date: 01/30/2024
+---
+# vcpkg_make_configure
+
+Configure a Make-based project.
+
+## Usage
+        
+```cmake
+vcpkg_make_configure(
+    SOURCE_PATH <source-path>
+    [AUTOCONFIG]
+    [COPY_SOURCE]
+    [DISABLE_MSVC_WRAPPERS]
+    [DISABLE_CPPFLAGS]
+    [DISABLE_DEFAULT_OPTIONS]
+    [DISABLE_MSVC_FLAG_ESCAPING]
+    [OPTIONS <configure-setting>...]
+    [OPTIONS_RELEASE <configure-setting>...]
+    [OPTIONS_DEBUG <configure-setting>...]
+    [PRE_CONFIGURE_CMAKE_COMMANDS <cmake-command>...]
+    [LANGUAGES <language>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### SOURCE_PATH
+
+The directory containing the project's source files.
+
+This value is usually obtained as a result of calling a source acquisition command like [`vcpkg_from_github()`](vcpkg_from_github.md).
+
+### AUTOCONFIG
+
+Runs autoreconf when set.
```

Comment:
Nit: Maybe it should be AUTORECONF instead? Not worth it if that would create serious upgrade pain.

Reply:
I'm in favor of the change,
FTR some packages prefer other methods (`bootstrap.sh`) for similar purposes.
FTR in the past, the keyword had the side effect of an implicit `DETERMINE_BUILD_TRIPLETS` which is hopefully removed here.